### DISCUSSION TITLE
How to login using Apple access relay

### DISCUSSION THREAD

**Discussion by TrkMstr25** [greghesp/ha-bambulab#678] [General]
Hello, I signed up months ago to Bambu Lab using Apple account and its private relay (hidden email) service. No problems at all logging to Bambu Lab websites, but how can I login from the HA integration?
I tried to use the private relay email and a new generated password, but it doesn't work.
Is there any workaround?

Thanks.

**Comment by kanflo** [comment]
BambuLab changed the logon process four days ago so it might be this issue you are facing, rather than an issue with the Apple relay email address. It is being addressed and should be resolved as soon as @greghesp finds the time.

**Comment by greghesp** [comment]
Did you try the alpha @AdrianGarside released a few days ago? It might solve this

**Comment by AdrianGarside** [comment]
Please try v2.0.28. I expect this is now fixed.
